tests/test_endless_data_attack.py

```diff
@@ -269,8 +269,8 @@ def test_with_tuf(self):
       self.repository_updater.refresh()
 
     except tuf.exceptions.NoWorkingMirrorError as exception:
-      for mirror_url, mirror_error in six.iteritems(exception.mirror_errors):
-        self.assertTrue(isinstance(mirror_error, securesystemslib.exceptions.Error))
+      for _, mirror_error in six.iteritems(exception.mirror_errors):
+        self.assertTrue(isinstance(mirror_error, tuf.exceptions.InvalidMetadataJSONError))
```

Comment:
Now, when all of the validation is done in `_verify_metadata_file` another type of exception will
be thrown for `timestamp.json` and thus I had to change it here.

Reply:
I think the old one is buggy and not clear.
I catch it in `_verify_metadata_file` and instead throw a more concrete exception.
I will leave comments on both places: where I catch it and in the test.